packages/sanes-chrome-extension/src/utils/test/persona.ts

```diff
@@ -0,0 +1,52 @@
+import { Store } from 'redux';
```

Comment:
The location of this file should be: `src/extension/background/model/persona/test` for modularity. I do not want to make huge a general folder.